trezor-crypto/include/TrezorCrypto/TrezorCrypto.h

```diff
@@ -1,29 +0,0 @@
-// Copyright © 2017-2020 Trust Wallet.
-//
-// This file is part of Trust. The full Trust copyright notice, including
-// terms governing use, modification, and redistribution, is contained in the
-// file LICENSE at the root of the source code distribution tree.
-
-#include <TrezorCrypto/address.h>
```

Comment:
can we keep this file? it's a umbrella header

Reply:
I've readded, but it is not used.